docs/app/Examples/elements/Segment/Types/SegmentVerticalSegmentExample.js

```diff
@@ -5,9 +5,9 @@ export default class SegmentVerticalSegmentExample extends Component {
   render() {
     return (
       <Segment>
-        <Segment className='vertical'>Te eum doming eirmod, nominati pertinacia argumentum ad his.</Segment>
-        <Segment className='vertical'>Pellentesque habitant morbi tristique senectus.</Segment>
-        <Segment className='vertical'>Eu quo homero blandit intellegebat. Incorrupte consequuntur mei id.</Segment>
+        <Segment aligned='vertical'>Te eum doming eirmod, nominati pertinacia argumentum ad his.</Segment>
+        <Segment aligned='vertical'>Pellentesque habitant morbi tristique senectus.</Segment>
+        <Segment aligned='vertical'>Eu quo homero blandit intellegebat. Incorrupte consequuntur mei id.</Segment>
```

Comment:
[Vertical](http://semantic-ui.com/elements/segment.html#vertical-segment) here applies to the vertical spacing and lack of vertical borders between consecutive segments.  I think this is another case of a boolean prop:

```js
<Segment vertical />
```
![image](https://cloud.githubusercontent.com/assets/5067638/16365137/27b1c6a4-3bac-11e6-83c7-0dc5afa169bd.png)

This is what makes sense to me at least.  Let me know if you disagree and why.